doc/source/user_guide/groupby.rst

```diff
@@ -183,7 +183,7 @@ By default the group keys are sorted during the ``groupby`` operation. You may h
 
 .. ipython:: python
 
-   df2 = pd.DataFrame({'X': ['B', 'B', 'A', 'A'], 'Y': [1, 2, 3, 4]})
+   df2 = pd.DataFrame({'X': ['A', 'A', 'B', 'B'], 'Y': [1, 2, 3, 4]})
```

Comment:
I think this was deliberate. We want the `X` column to be unsorted so that the two operations below (`sort=True/False`) are differentiations.

Reply:
Now reading it more carefully I can see the intention of this section. I will pay more attention when proposing small changes similar to that one, sorry.